internal/provider/client.go

```diff
@@ -92,6 +92,7 @@ func (c *Client) ImportStory(ctx context.Context, storyData []byte) (sr StoryRes
 	}
 	req.Header.Set("content-type", "application/json")
 	req.Header.Add("Accept", "application/json")
+	req.Header.Set("User-Agent", "tines-terraform-client")
```

Comment:
worth adding a version here or in an extra header? might be useful for debugging/feature-flagging/etc

Reply:
yeah good call, updated